python/vowpalwabbit/DFtoVWtoTensorboard.py

```diff
@@ -0,0 +1,182 @@
+		vw		 : A vowpalwabbit object
+ 					This would be used for learning using the algorithm specified in the object, and also to compute a few metrics
+
+		Returns
+		-------
+
+		self : DFtoVWtoTensorboard
+		"""
+		self.vw = vw
+		self.vw_formatted_data = df_to_vw.convert_df()
+
+
+	#------------------------------------------------------------------------------------------------		
+	def _calculate_average_loss(self, sum_loss, weighted_examples):
+		"""Calculate average loss
+
+		Parameters
+		----------
+
+		sum_loss			: float
+								This is the sum loss which is used to calculate average_loss
+		weighted_examples   : float
+								This is the weighted_example which is used to calculate average_loss
+
+		Returns
+		-------
+
+		average_loss      : float
+								average_loss calculated by sum_loss and weigted_examples or if ZeroDivisionError is encountered then 0. is returned 
+		""" 
+		try:
+			return sum_loss / weighted_examples
+
+		except ZeroDivisionError:
+			return 0.
+
+
+	#------------------------------------------------------------------------------------------------		
+	def _calculate_since_last(self, sum_loss_since_last, weighted_examples_since_last):
+		"""Calculate since last 
+
+		Parameters
+		----------
+
+		sum_loss_since_last          : float
+										This is the difference between current iteration's sum_loss and previous iteration's sum_loss which is used to calculate since_last
+		weighted_examples_since_last : float
+										This is the difference between current iteration's weighted_examples and previous iteration's weighted_examples which is used to calculate since_last
+
+		Returns
+		-------
+
+		since_last      : float
+							since_last calculated using sum_loss_since_last and weighted_examples_since_last or if ZeroDivisionError is encountered then 0. is returned 
+		""" 
+		try:
+			return sum_loss_since_last / weighted_examples_since_last
+
+		except ZeroDivisionError:
+			return 0.
+
+
+	#------------------------------------------------------------------------------------------------		
+	def _print_metrics(self, average_loss, since_last, label, prediction, num_features):
+		"""This method prints metrics
+
+		Parameters
+		----------
+
+		average_loss : float
+						This is the average_loss value
+		since_last   : float
+						This is the since_last value
+		label        : integer
+						This is the actual label of current example
+		prediction   : integer
+						This is the predicted label of current example
+		num_features : integer
+						This is the num_features value
+
+		Returns
+		-------
+
+		None
+		"""
+		print( 'average_loss:{:.6f}'.format(average_loss) , end='\t')
+		print('since_last:{:.6f}'.format(since_last), end='\t')
+		print('label:', label, end='\t')    
+		print('prediction:', prediction, end='\t')    
+		print('num_features:', num_features) 
+
+
+	#------------------------------------------------------------------------------------------------		
+	def fit(self, vw_to_tensorboard=None):
```

Comment:
Consider merging this class with the pyvw class, in particular, pulling code in the learning loop here into `pyvw.learn`: This will allow everyone who uses pyVW to use your VWtoTensorboard integration, not only those using DFtoVW.